xbrowse_server/phenotips/utilities.py

```diff
@@ -0,0 +1,121 @@
+import os
+import logging
+from django.conf import settings
+import json
+import urllib2
+import base64
+import requests
+logger = logging.getLogger(__name__)
+import sys
+
+#create a patient record in phenotips
+#by convention username and password are project_id,project_idproject_id
```

Comment:
Updated and committed. This is a debate I have with myself all the time. Doc tags vs method comments. In the past I have done both on the same method, thinking method comments are for active developers and doc strings being for developers simply importing in the code as a library. That turned a bit ugly with mostly the same comment appearing twice, so I sort of tried out method comments only. I think if you prefer doc strings, we can stick to that exclusively since that would be the best of both worlds (library users and active developers).